src/_includes/hubspot_cta.liquid

```diff
@@ -0,0 +1,19 @@
+{%- if site.lang == 'es' -%}
+  <span class="{{ include.class }}" id="hs-cta-wrapper-{{ include.hubspot-id }}">
+    <span class="hs-cta-node hs-cta-{{ include.hubspot-id }}" id="hs-cta-{{ include.hubspot-id }}">
+      <!--[if lte IE 8]><div id="hs-cta-ie-element"></div><![endif]-->
+      <a href="https://cta-redirect.hubspot.com/cta/redirect/3042464/{{ include.hubspot-id }}" ></a>
+    </span>
+    <script charset="utf-8" src="https://js.hscta.net/cta/current.js"></script>
+    <script type="text/javascript"> hbspt.cta.load(3042464, '{{ include.hubspot-id }}', {}); </script>
+  </span>
+{%- else -%}
+  <span class="{{ include.class }}" id="hs-cta-wrapper-{{ include.hubspot-id }}">
+    <span class="hs-cta-node hs-cta-{{ include.hubspot-id }}" id="hs-cta-{{ include.hubspot-id }}">
+      <!--[if lte IE 8]><div id="hs-cta-ie-element"></div><![endif]-->
+      <a href="https://cta-redirect.hubspot.com/cta/redirect/3042464/{{ include.hubspot-id }}" ></a>
+    </span>
+    <script charset="utf-8" src="https://js.hscta.net/cta/current.js"></script>
+    <script type="text/javascript"> hbspt.cta.load(3042464, '{{ include.hubspot-id }}', {}); </script>
+  </span>
+{%- endif -%}
```

Comment:
Do we need the if else? Both sides of the conditional look the same

Reply:
True! I guess I never came back for this. Amended here: [4dae82d](https://github.com/codurance/site/pull/1849/commits/4dae82ddc3b2de1c3c8cfc1efa47b1e6e253355f)